### ISSUE TITLE
Waiting for iTunes Connect to finish processing the new build 

### ISSUE THREAD

**Issue by RishabhTayal** [fastlane/fastlane#9664]
### New Issue Checklist

- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] Updated fastlane to the latest version

### Issue Description

I am using `testflight` to upload the build to TestFlight and submit it for external distribution. My build on iTunes connect has finished processing and available for Internal testers, while the `fastlane` script is still stuck for a while on `Waiting for iTunes Connect to finish processing the new build `.

Any solutions?

### Environment
<details><summary>✅ fastlane environment ✅</summary>

### Stack

| Key                         | Value                                            |
| --------------------------- | ------------------------------------------------ |
| OS                          | 10.12.5                                          |
| Ruby                        | 2.2.4                                            |
| Bundler?                    | false                                            |
| Git                         | git version 2.13.0 (Apple Git-88)                |
| Installation Source         | ~/.fastlane/bin/bundle/bin/fastlane              |
| Host                        | Mac OS X 10.12.5 (16F73)                         |
| Ruby Lib Dir                | ~/.fastlane/bin/bundle/lib                       |
| OpenSSL Version             | OpenSSL 1.0.2g  1 Mar 2016                       |
| Is contained                | true                                             |
| Is homebrew                 | false                                            |
| Is installed via Fabric.app | false                                            |
| Xcode Path                  | /Applications/Xcode-beta.app/Contents/Developer/ |
| Xcode Version               | 9.0                                              |

### System Locale

| Variable | Value       |   |
| -------- | ----------- | - |
| LANG     | en_US.UTF-8 | ✅ |
| LC_ALL   | en_US.UTF-8 | ✅ |
| LANGUAGE | en_US.UTF-8 | ✅ |

### fastlane files:

<details><summary>`./fastlane/Fastfile`</summary>

```ruby
fastlane_version "2.36.0"

default_platform :ios

ENV['DELIVER_ITMSTRANSPORTER_ADDITIONAL_UPLOAD_PARAMETERS'] = '-t DAV'

update_fastlane

platform :ios do

  desc "\033[1mDeploy to iTunes Connect\033[0m"
  desc "This action does the following:"
  desc "- Input version number from user"
  desc "- Increament version number"
  desc "- Increament build number"
  desc "- Generate build"
  desc "- Push the changed plist to git remote"
  desc "- Upload build to TestFlight"
  desc "- Upload dSYM files to Hockey"
  lane :deploy_itunes do |options|
    build_number = sh("curl <Redacted> --silent")
    deploy_app_appstore('<Redacted>', '<Redacted>', options[:increament_version_number], build_number, options[:submit_for_review], "<Redacted>")
    refresh_dsyms
  end

  def deploy_app_appstore(scheme_name, xcodeproj_path, increament_version_number, build_number, submit_for_review, app_identifier)
   if increament_version_number == nil
    increament_version_number = prompt(text: "Do you want to increase the version number? Next time you can run `fastlane " + lane_context[SharedValues::LANE_NAME] + " increament_version_number:false`", ci_input: increament_version_number, boolean: true)
   end
   if !build_number
    build_number = prompt(text: "Enter build number", ci_input: build_number)
   end
   if increament_version_number == true
     increment_version_number(
      bump_type: "patch",
      xcodeproj: xcodeproj_path
      )
  end
   build_number = increment_build_number(
    build_number: build_number,
    xcodeproj: xcodeproj_path
    )
   gym({
    scheme: scheme_name,
    export_method: 'app-store',
    workspace: '<Redacted>.xcworkspace',
    clean: true,
    include_bitcode: false,
    codesigning_identity: '<Redacted>',
    silent: true
    })
    commit_push_to_master_develop_git('<Redacted>/v' + build_number)
    app_description = File.read("app_description.txt")
   testflight(
    username: '<Redacted>',
    skip_waiting_for_build_processing: false,
    distribute_external: true,
    wait_processing_interval: 3600,
    beta_app_description: app_description, 
    changelog: "Change log", 
    beta_app_feedback_email: "<Redacted>",
    groups: '<Redacted>'
    )
   if submit_for_review == nil
    submit_for_review = prompt(text: "Do you want to submit for App Store review? Next time you can run `fastlane " + lane_context[SharedValues::LANE_NAME] + " submit_for_review:false`", ci_input: submit_for_review, boolean: true)
   end
   if submit_for_review == true
    deliver(force: true, submit_for_review: true, skip_binary_upload: true, skip_screenshots: true, skip_metadata: true, app_identifier: app_identifier)
   end
 end

lane :default_changelog do
  changelog = changelog_from_git_commits(
    between: [ENV['GIT_PREVIOUS_SUCCESSFUL_COMMIT'] || "HEAD^^^^^", "HEAD"],
    pretty: "- %s via %aN",
    merge_commit_filtering: 'only_include_merges'
    )
  puts("Filtering the change log...")
  changelog = changelog.sub(/[\u{1F300}-\u{1F6FF}]/, '')
  changelog = changelog.gsub(/(.*)Merge branch \'develop\'(.*)/, '')
  changelog = changelog.gsub(/(.*)Merge branch \'master\'(.*)/, '')
  changelog = changelog.gsub(/\n{2,}/, "\n")
  puts("Filtered change log:" + changelog)
  Actions.lane_context[SharedValues::FL_CHANGELOG] = changelog
  changelog
end

private_lane :refresh_dsyms do
  hockey(
    api_token: '<Redacted>',
    dsym: '<Redacted>.app.dSYM.zip',
    upload_dsym_only: true,
    public_identifier: "<Redacted>"
    )
end

def commit_push_to_master_develop_git(git_tag)
  git_commit ({
    path: ".",
    message: 'Build Version Bump by fastlane [ci skip]'
    })
  git_pull
  add_git_tag(tag: git_tag)
  push_to_git_remote
  sh "git checkout develop && git pull && git merge master -s ours && git push && git checkout master"
end

end
```
</details>

<details><summary>`./fastlane/Appfile`</summary>

```ruby
app_identifier "" # The bundle identifier of your app
apple_id "" # Your Apple email address

# You can uncomment any of the lines below and add your own
# team selection in case you're in multiple teams
team_name "<Redacted>"
# team_id "<Redacted>"

# To select a team for iTunes Connect use
itc_team_name "<Redacted>"

# you can even provide different app identifiers, Apple IDs and team names per lane:
# https://github.com/KrauseFx/fastlane/blob/master/docs/Appfile.md
```
</details>

### fastlane gems

| Gem      | Version | Update-Status |
| -------- | ------- | ------------- |
| fastlane | 2.45.0  | ✅ Up-To-Date  |

### Loaded fastlane plugins:

**No plugins Loaded**

<details><summary><b>Loaded gems</b></summary>

| Gem                                | Version      |
| ---------------------------------- | ------------ |
| CFPropertyList                     | 2.3.5        |
| activesupport                      | 4.2.8        |
| addressable                        | 2.5.1        |
| babosa                             | 1.0.2        |
| bigdecimal                         | 1.2.6        |
| bundler                            | 1.14.6       |
| claide                             | 1.0.1        |
| clamp                              | 0.6.5        |
| cocoapods                          | 1.2.1        |
| cocoapods-core                     | 1.2.1        |
| cocoapods-deintegrate              | 1.0.1        |
| cocoapods-downloader               | 1.1.3        |
| cocoapods-plugins                  | 1.0.0        |
| cocoapods-search                   | 1.0.0        |
| cocoapods-stats                    | 1.0.0        |
| cocoapods-trunk                    | 1.2.0        |
| cocoapods-try                      | 1.1.0        |
| colored                            | 1.2          |
| colored2                           | 3.1.2        |
| colorize                           | 0.8.1        |
| commander-fastlane                 | 4.4.5        |
| commander-fastlane                 | 4.4.4        |
| cork                               | 0.3.0        |
| declarative                        | 0.0.9        |
| declarative-option                 | 0.1.0        |
| domain_name                        | 0.5.20170404 |
| dotenv                             | 2.2.1        |
| dotenv                             | 2.2.0        |
| escape                             | 0.0.4        |
| excon                              | 0.55.0       |
| faraday                            | 0.12.1       |
| faraday                            | 0.12.0.1     |
| faraday-cookie_jar                 | 0.0.6        |
| faraday_middleware                 | 0.11.0.1     |
| fastimage                          | 2.1.0        |
| fastlane-plugin-android_versioning | 0.2.2        |
| fastlane-plugin-changelog          | 0.7.0        |
| fastlane-plugin-clubmate           | 0.1.0        |
| fastlane-plugin-ruby               | 0.1.3        |
| fastlane-plugin-sharethemeal       | 0.1.9        |
| fastlane-plugin-slack_train        | 0.2.0        |
| fastlane-plugin-synx               | 0.1.3        |
| fourflusher                        | 2.0.1        |
| fuzzy_match                        | 2.0.4        |
| gh_inspector                       | 1.0.3        |
| git                                | 1.3.0        |
| google-api-client                  | 0.12.0       |
| google-api-client                  | 0.9.28       |
| googleauth                         | 0.5.1        |
| highline                           | 1.7.8        |
| http-cookie                        | 1.0.3        |
| httpclient                         | 2.8.3        |
| hurley                             | 0.2          |
| i18n                               | 0.8.1        |
| io-console                         | 0.4.3        |
| json                               | 1.8.1        |
| jwt                                | 1.5.6        |
| kramdown                           | 1.13.2       |
| libxml-ruby                        | 3.0.0        |
| little-plugger                     | 1.1.4        |
| logging                            | 2.2.2        |
| memoist                            | 0.15.0       |
| mime-types                         | 3.1          |
| mime-types-data                    | 3.2016.0521  |
| mini_magick                        | 4.5.1        |
| mini_portile2                      | 2.2.0        |
| mini_portile2                      | 2.1.0        |
| minitest                           | 5.4.3        |
| molinillo                          | 0.5.7        |
| multi_json                         | 1.12.1       |
| multi_xml                          | 0.6.0        |
| multipart-post                     | 2.0.0        |
| nanaimo                            | 0.2.3        |
| nap                                | 1.1.0        |
| netrc                              | 0.11.0       |
| netrc                              | 0.7.8        |
| nokogiri                           | 1.5.6        |
| open4                              | 1.3.4        |
| os                                 | 0.9.6        |
| plist                              | 3.3.0        |
| plist                              | 3.2.0        |
| power_assert                       | 0.2.2        |
| psych                              | 2.0.8        |
| public_suffix                      | 2.0.5        |
| rake                               | 10.4.2       |
| rdoc                               | 4.2.0        |
| representable                      | 3.0.4        |
| representable                      | 2.3.0        |
| rest-client                        | 2.0.2        |
| retriable                          | 2.1.0        |
| rouge                              | 2.0.7        |
| rouge                              | 1.11.1       |
| ruby-macho                         | 1.1.0        |
| rubygems-update                    | 2.5.2        |
| rubyzip                            | 1.2.1        |
| security                           | 0.1.3        |
| signet                             | 0.7.3        |
| slack-notifier                     | 1.5.1        |
| sqlite3                            | 1.3.13       |
| stm_api                            | 0.1.7        |
| synx                               | 0.2.1        |
| terminal-notifier                  | 1.7.1        |
| terminal-table                     | 1.7.3        |
| test-unit                          | 3.0.8        |
| thread_safe                        | 0.3.6        |
| tty-screen                         | 0.5.0        |
| tzinfo                             | 1.2.3        |
| uber                               | 0.0.15       |
| unf                                | 0.1.4        |
| unf_ext                            | 0.0.7.4      |
| unicode-display_width              | 1.1.3        |
| word_wrap                          | 1.0.0        |
| xcodeproj                          | 1.4.4        |
| xcpretty                           | 0.2.6        |
| xcpretty-travis-formatter          | 0.0.4        |
</details>

*generated on:* **2017-07-06**
</details>

### Output

<details><summary>Output</summary>

```
+ export LANG=en_US.UTF-8
+ LANG=en_US.UTF-8
+ export LANGUAGE=en_US.UTF-8
+ LANGUAGE=en_US.UTF-8
+ export LC_ALL=en_US.UTF-8
+ LC_ALL=en_US.UTF-8
+ /Users/jenkins/.rbenv/shims/pod install
Analyzing dependencies
Downloading dependencies
Using ARChromeActivity (1.0.5)
Integrating client projects
Pod installation complete! There are 37 dependencies from the Podfile and 45 total pods installed.
+ /Users/jenkins/.rbenv/shims/fastlane deploy_itunes increament_version_number:false submit_for_review:true
[15:00:16]: -------------------------------------------------
[15:00:16]: --- Step: Verifying required fastlane version ---	
[15:00:16]: -------------------------------------------------
[15:00:16]: Your fastlane version 2.45.0 matches the minimum requirement of 2.36.0  ✅
[15:00:16]: ------------------------------
[15:00:16]: --- Step: default_platform ---
[15:00:16]: ------------------------------
[15:00:16]: -----------------------------
[15:00:16]: --- Step: update_fastlane ---
[15:00:16]: -----------------------------
[15:00:16]: Looking for updates for fastlane...
[15:00:19]: Nothing to update ✅
[15:00:19]: 
[15:00:19]: Please help us test early releases of fastlane by opting into nightly builds 🌃
[15:00:19]: Just replace your `update_fastlane` call with
[15:00:19]: 
[15:00:19]: ▸ update_fastlane(nightly: true)
[15:00:19]: 
[15:00:19]: Nightly builds are reviewed and tested just like the public releases 🚂
[15:00:19]: 
[15:00:19]: -------------------------------------------------
[15:00:19]: --- Step: Verifying required fastlane version ---
[15:00:19]: -------------------------------------------------
[15:00:19]: Your fastlane version 2.45.0 matches the minimum requirement of 2.28.4  ✅
[15:00:19]: ------------------------------
[15:00:19]: --- Step: default_platform ---
[15:00:19]: ------------------------------
[15:00:19]: -------------------------------------------------
[15:00:19]: --- Step: Verifying required fastlane version ---
[15:00:19]: -------------------------------------------------
[15:00:19]: Your fastlane version 2.45.0 matches the minimum requirement of 2.2.0  ✅
[15:00:19]: ------------------------------
[15:00:19]: --- Step: default_platform ---
[15:00:19]: ------------------------------
[15:00:19]: Driving the lane 'ios deploy_itunes' 🚀
[15:00:19]: --------------------------------------------------------------------
[15:00:19]: Step: curl <Redacted> --silent
[15:00:19]: --------------------------------------------------------------------
[15:00:19]: $ curl <Redacted> --silent
[15:00:19]: ------------------------------------
[15:00:19]: --- Step: increment_build_number ---
[15:00:19]: ------------------------------------
Current version of project <Redacted> is: 

/Users/jenkins/.jenkins/workspace/<Redacted>
[15:00:19]: $ cd /Users/jenkins/.jenkins/workspace/<Redacted>/<Redacted> && agvtool new-version -all 434 && cd -
[15:00:19]: ▸ Setting version of project <Redacted> to:
[15:00:19]: ▸ Also setting CFBundleVersion key (assuming it exists)
[15:00:19]: ▸ Updating CFBundleVersion in Info.plist(s)...
[15:00:19]: ▸ /Users/jenkins/.jenkins/workspace/<Redacted>
[15:00:20]: -----------------
[15:00:20]: --- Step: gym ---
[15:00:20]: -----------------
[15:00:20]: Detected provisioning profile mapping: {<Redacted>}
[15:00:20]: $ xcodebuild -list -workspace <Redacted>
[15:00:22]: $ xcodebuild -showBuildSettings -workspace <Redacted> -scheme <Redacted>

+--------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
|                                                                                 Summary for gym 2.44.1                                                                                  |
+--------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+
| scheme                                                                                                 | <Redacted>                                                              |
| export_method                                                                                          | app-store                                                                      |
| workspace                                                                                              | <Redacted>                                                    |
| clean                                                                                                  | true                                                                           |
| include_bitcode                                                                                        | false                                                                          |
| codesigning_identity                                                                                   | <Redacted>                                      |
| silent                                                                                                 | true                                                                           |
<Redacted>
| destination                                                                                            | generic/platform=iOS                                                           |
| output_name                                                                                            | <Redacted>                                                                       |
| build_path                                                                                             | /Users/jenkins/Library/Developer/Xcode/Archives/2017-07-03                     |
| output_directory                                                                                       | .                                                                              |
| skip_package_ipa                                                                                       | false                                                                          |
| buildlog_path                                                                                          | ~/Library/Logs/gym                                                             |
| xcode_path                                                                                             | /Applications/Xcode.app                                                        |
+--------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------+

<Redacted>
[15:05:35]: ▸ Compiling Main.storyboard
[15:05:35]: ▸ Processing Info.plist
[15:05:35]: ▸ Generating '<Redacted>.app.dSYM'
[15:05:36]: ▸ Running script 'TODO Script'
[15:05:36]: ▸ Running script '[CP] Embed Pods Frameworks'
[15:05:44]: ▸ Running script '[CP] Copy Pods Resources'
[15:05:44]: ▸ Copying /Users/jenkins/Library/Developer/Xcode/DerivedData/<Redacted>-dikewuhbucoxaxbvatxscwhsgktz/Build/Intermediates/ArchiveIntermediates/<Redacted>/BuildProductsPath/AppStore-iphoneos/Common.framework
[15:05:44]: ▸ Signing /Users/jenkins/Library/Developer/Xcode/DerivedData/<Redacted>-dikewuhbucoxaxbvatxscwhsgktz/Build/Intermediates/ArchiveIntermediates/<Redacted>/InstallationBuildProductsLocation/Applications/<Redacted>.app/Frameworks/Common.framework
[15:05:45]: ▸ Touching <Redacted>.app
[15:05:47]: ▸ Signing /Users/jenkins/Library/Developer/Xcode/DerivedData/<Redacted>-dikewuhbucoxaxbvatxscwhsgktz/Build/Intermediates/ArchiveIntermediates/<Redacted>/InstallationBuildProductsLocation/Applications/<Redacted>.app
[15:05:48]: ▸ Touching <Redacted>.app.dSYM
[15:05:49]: ▸ Archive Succeeded
[15:05:49]: Generated plist file with the following values:
[15:05:49]: ▸ -----------------------------------------
[15:05:49]: ▸ {
[15:05:49]: ▸   "provisioningProfiles": {
[15:05:49]: ▸     "<Redacted>"
[15:05:49]: ▸   },
[15:05:49]: ▸   "method": "app-store",
[15:05:49]: ▸   "uploadBitcode": false
[15:05:49]: ▸ }
[15:05:49]: ▸ -----------------------------------------

[15:06:45]: Successfully exported and compressed dSYM file
[15:06:45]: Successfully exported and signed the ipa file:
[15:06:45]: /Users/jenkins/.jenkins/workspace/<Redacted>
[15:06:45]: ------------------------
[15:06:45]: --- Step: git_commit ---
[15:06:45]: ------------------------
[15:06:45]: $ git commit -m Build\ Version\ Bump\ by\ fastlane\ \[ci\ skip\] .
[15:06:46]: ▸ [master 1e85510e8] Build Version Bump by fastlane [ci skip]
[15:06:46]: ▸ 6 files changed, 8 insertions(+), 8 deletions(-)
[15:06:46]: Successfully committed "." 💾.
[15:06:46]: ----------------------
[15:06:46]: --- Step: git_pull ---
[15:06:46]: ----------------------
[15:06:46]: $ git pull && git fetch --tags
[15:06:47]: ▸ From <Redacted>
[15:06:47]: ▸ 2d7373ab9..ba5bbaad3  develop    -> origin/develop
[15:06:47]: ▸ Already up-to-date.
[15:06:47]: -------------------------
[15:06:47]: --- Step: add_git_tag ---
[15:06:47]: -------------------------
[15:06:47]: Adding git tag '<Redacted>/v434' 🎯.
[15:06:47]: $ git tag -am <Redacted>/v434\ \(fastlane\) '<Redacted>/v434'
[15:06:47]: --------------------------------
[15:06:47]: --- Step: push_to_git_remote ---
[15:06:47]: --------------------------------
[15:06:47]: $ pwd
[15:06:47]: ▸ /Users/jenkins/.jenkins/workspace/<Redacted>
[15:06:47]: $ git push origin master:master --tags
[15:06:48]: ▸ To <Redacted>
[15:06:48]: ▸ 12a736b60..1e85510e8  master -> master
[15:06:48]: ▸ * [new tag]             <Redacted>/v434 -> <Redacted>/v434
[15:06:48]: Successfully pushed to remote.
[15:06:48]: --------------------------------------------------------------------
[15:06:48]: Step: git checkout develop && git pull && git merge master -s ours && git push && git checkout master
[15:06:48]: --------------------------------------------------------------------
[15:06:48]: $ git checkout develop && git pull && git merge master -s ours && git push && git checkout master
[15:06:48]: ▸ Switched to branch 'develop'
[15:06:48]: ▸ Your branch is behind 'origin/develop' by 11 commits, and can be fast-forwarded.
[15:06:48]: ▸ (use "git pull" to update your local branch)
[15:06:48]: ▸ Updating 9d5a6ca9a..ba5bbaad3
[15:06:48]: ▸ Fast-forward
[15:06:48]: ▸ <Redacted>/<Redacted>/<Redacted>.storyboard | 14 ++--
[15:06:48]: ▸ fastlane/Fastfile                                  |  5 +-
[15:06:48]: ▸ 12 files changed, 118 insertions(+), 60 deletions(-)
[15:06:48]: ▸ create mode 100644 fastlane/<Redacted>
[15:06:48]: ▸ create mode 100644 fastlane/<Redacted>
[15:06:48]: ▸ Merge made by the 'ours' strategy.
[15:06:49]: ▸ remote:
[15:06:49]: ▸ remote: Create pull request for develop:
[15:06:49]: ▸ remote:   <Redacted>
[15:06:49]: ▸ remote:
[15:06:49]: ▸ To <Redacted>
[15:06:49]: ▸ ba5bbaad3..cadb6e650  develop -> develop
[15:06:49]: ▸ Switched to branch 'master'
[15:06:49]: ▸ Your branch is up-to-date with 'origin/master'.
[15:06:49]: ------------------------
[15:06:49]: --- Step: testflight ---
[15:06:49]: ------------------------
[15:06:49]: Login to iTunes Connect (<Redacted>)
[15:06:52]: Login successful
[15:06:53]: Ready to upload new build to TestFlight (App: <Redacted>)...
[15:06:54]: Going to upload updated app to iTunes Connect
[15:06:54]: This might take a few minutes. Please don't interrupt the script.
[15:13:04]: iTunes Transporter successfully finished its job
[15:13:04]: --------------------------------------------------------------------
[15:13:04]: Successfully uploaded package to iTunes Connect. It might take a few minutes until it's visible online.
[15:13:04]: --------------------------------------------------------------------
[15:13:04]: Successfully uploaded the new binary to iTunes Connect
[15:13:04]: If you want to skip waiting for the processing to be finished, use the `skip_waiting_for_build_processing` option
[15:14:10]: Waiting for iTunes Connect to finish processing the new build (0.94.77 - 434)
[16:14:12]: Build <Redacted> is already being tested
[16:14:14]: Successfully set the beta_app_feedback_email and/or beta_app_description
[16:14:17]: Successfully set the changelog for build
[16:14:17]: Distributing new build to testers: 0.94.77 - 434
+--------------------+---------------------------------------------------------------------------------------------------+
|                                                      Lane Context                                                      |
+--------------------+---------------------------------------------------------------------------------------------------+
| DEFAULT_PLATFORM   | ios                                                                                               |
| PLATFORM_NAME      | ios                                                                                               |
| LANE_NAME          | ios <Redacted>                                                                              |
| BUILD_NUMBER       | 434                                                                                               |
| IPA_OUTPUT_PATH    | /Users/jenkins/.jenkins/workspace/<Redacted>                           |
| DSYM_OUTPUT_PATH   | /Users/jenkins/.jenkins/workspace/<Redacted>.app.dSYM.zip                  |
| XCODEBUILD_ARCHIVE | /Users/jenkins/Library/Developer/Xcode/Archives/2017-07-03/<Redacted> 2017-07-03 15.00.25.xcarchive |
+--------------------+---------------------------------------------------------------------------------------------------+
[16:14:18]: {"code"=>"ITC.response.error.INVALID_REQUEST", "description"=>"Failed to submit build for review."}

+------+---------------------------------------------------------------+-------------+
|                                  fastlane summary                                  |
+------+---------------------------------------------------------------+-------------+
| Step | Action                                                        | Time (in s) |
+------+---------------------------------------------------------------+-------------+
| 1    | Verifying required fastlane version                           | 0           |
| 2    | default_platform                                              | 0           |
| 3    | update_fastlane                                               | 2           |
| 4    | Verifying required fastlane version                           | 0           |
| 5    | default_platform                                              | 0           |
| 6    | Verifying required fastlane version                           | 0           |
| 7    | default_platform                                              | 0           |
| 8    | curl <Redacted> | 0           |
| 9    | increment_build_number                                        | 1           |
| 10   | gym                                                           | 385         |
| 11   | git_commit                                                    | 0           |
| 12   | git_pull                                                      | 1           |
| 13   | add_git_tag                                                   | 0           |
| 14   | push_to_git_remote                                            | 0           |
| 15   | git checkout develop && git pull && git merge master -s ours  | 1           |
| 💥   | testflight                                                    | 4049        |
+------+---------------------------------------------------------------+-------------+

[16:14:18]: fastlane finished with errors

Looking for related GitHub issues on fastlane/fastlane...

Found no similar issues. To create a new issue, please visit:
https://github.com/fastlane/fastlane/issues/new
Run `fastlane env` to append the fastlane environment to your issue
/Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/spaceship/lib/spaceship/test_flight/client.rb:200:in `handle_response': [!] {"code"=>"ITC.response.error.INVALID_REQUEST", "description"=>"Failed to submit build for review."} (Spaceship::Client::UnexpectedResponse)
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/spaceship/lib/spaceship/test_flight/client.rb:68:in `post_for_testflight_review'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/spaceship/lib/spaceship/test_flight/build.rb:179:in `submit_for_testflight_review!'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/pilot/lib/pilot/build_manager.rb:139:in `distribute_build'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/pilot/lib/pilot/build_manager.rb:76:in `distribute'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/pilot/lib/pilot/build_manager.rb:40:in `upload'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/actions/pilot.rb:16:in `run'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/actions/testflight.rb:5:in `run'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:252:in `block (2 levels) in execute_action'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/actions/actions_helper.rb:50:in `execute_action'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:230:in `block in execute_action'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:226:in `chdir'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:226:in `execute_action'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:148:in `trigger_action_by_name'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/fast_file.rb:146:in `method_missing'
	from Fastfile:200:in `deploy_app_appstore'
	from Fastfile:59:in `block (2 levels) in parsing_binding'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/lane.rb:33:in `call'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:49:in `block in execute'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:45:in `chdir'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/runner.rb:45:in `execute'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/lane_manager.rb:52:in `cruise_lane'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/command_line_handler.rb:30:in `handle'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/commands_generator.rb:104:in `block (2 levels) in run'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:178:in `call'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/command.rb:153:in `run'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/runner.rb:476:in `run_active_command'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb:39:in `run!'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/commander-fastlane-4.4.5/lib/commander/delegates.rb:15:in `run!'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/commands_generator.rb:303:in `run'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/commands_generator.rb:42:in `start'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/fastlane/lib/fastlane/cli_tools_distributor.rb:66:in `take_off'
	from /Users/jenkins/.rbenv/versions/2.2.6/lib/ruby/gems/2.2.0/gems/fastlane-2.44.1/bin/fastlane:20:in `<top (required)>'
	from /Users/jenkins/.rbenv/versions/2.2.6/bin/fastlane:22:in `load'
	from /Users/jenkins/.rbenv/versions/2.2.6/bin/fastlane:22:in `<main>'
Build step 'Execute shell' marked build as failure
Finished: FAILURE
```
</details>

**Comment by pav1790** [comment]
Could you share the environment. I think I saw this issue only once in the one of the previous builds in my own use of fastlane. Not seeing it in any of the latest ones. Could be a hidden bug too.

**Comment by KrauseFx** [comment]
Yeah, please share your environment and the complete output :+1:

**Comment by RishabhTayal** [comment]
Added environment to the description.

**Comment by KrauseFx** [comment]
Please share your complete output when running _fastlane_

**Comment by RishabhTayal** [comment]
Added.

**Comment by KrauseFx** [comment]
Sorry, but that's just the stack trace, we need the complete output

**Comment by RishabhTayal** [comment]
@KrauseFx Updated the description with complete output.

**Comment by KrauseFx** [comment]
Thanks, that's interesting

```
[16:14:18]: {"code"=>"ITC.response.error.INVALID_REQUEST", "description"=>"Failed to submit build for review."}
```

Could you try manually submitting the following build for TestFlight review: `0.94.77 - 434`

**Comment by RishabhTayal** [comment]
When the Fastlane script failed I was able to sumit the build manually for review.

**Comment by RishabhTayal** [comment]
Any update on this?

**Comment by KrauseFx** [comment]
Sorry, no. This seems like something we'll have a hard time reproducing. If you can tell us what's the best way to reliable reproduce the problem, we can look into it, otherwise it's really difficult to work on a fix 😢

**Comment by RishabhTayal** [comment]
Thanks for getting back. I don't think there is anything different I am doing to reproduce this problem. I have the standard setup for lane. 

The best way to reproduce this issue is to just upload the ipa through `pilot` and submit that build to app store review with `deliver`. 

I wish I could have more information to provide.

**Comment by RishabhTayal** [comment]
Sometimes it just hangs on 'waiting for iTunes Connect to finish processing' for ever even though the build is completed processing on iTC. Here is the output.

```
[13:18:25]: Ready to upload new build to TestFlight (App: xxxx)...
[13:18:25]: Going to upload updated app to iTunes Connect
[13:18:25]: This might take a few minutes. Please don't interrupt the script.
[13:24:40]: iTunes Transporter successfully finished its job
[13:24:40]: --------------------------------------------------------------------
[13:24:40]: Successfully uploaded package to iTunes Connect. It might take a few minutes until it's visible online.
[13:24:40]: --------------------------------------------------------------------
[13:24:40]: Successfully uploaded the new binary to iTunes Connect
[13:24:40]: If you want to skip waiting for the processing to be finished, use the `skip_waiting_for_build_processing` option
[13:25:01]: Waiting for iTunes Connect to finish processing the new build (0.94.78 - 456)
```
